recipes/iperf/iperf_3.1.3.oe

```diff
@@ -0,0 +1,3 @@
+require iperf.inc
+
+CFLAGS=""
```

Comment:
I can't remember why... I have fixed it, let's see what the builder says without it :smile: